docs/getting-started.md

```diff
@@ -5,33 +5,36 @@ There are 2 ways of deploying Sleeper and interacting with an instance. You can
 local machine. The Docker version has limited functionality and will only work with small volumes of data, but will
 allow you to deploy an instance, ingest some files, and run reports and scripts against the instance.
 
-## Deploy to Docker
+## Sleeper in LocalStack
 
 The quickest way to get an instance of Sleeper is to deploy to LocalStack in Docker on your local machine. Note that the
 LocalStack version has very limited functionality in comparison to the AWS version, and can only handle small volumes of
-data. See the documentation on [deploying to localstack](usage/deploy-to-localstack.md) for more information.
+data. See the documentation on [deploying to localstack](deployment/deploy-to-localstack.md) for more information.
 
-## Deploy to AWS
+The rest of this guide will deal with Sleeper in AWS.
 
-The easiest way to deploy a full instance of Sleeper and interact with it is to use the "system test" functionality.
-This deploys a Sleeper instance with a simple schema, and writes some random data into a table in the instance. You can
-then use the status scripts to see how much data is in the system, run some example queries, and view logs to help
-understand what the system is doing. It is best to do this from an EC2 instance as a significant amount of code needs to
-be uploaded to AWS.
+## Sleeper in AWS
+
+This Git repository contains scripts that let you build and/or deploy Sleeper with a single command, to minimise setup.
+The Sleeper CLI lets you run these scripts in a Docker container, with only Docker as a pre-installed dependency.
+
+It's currently necessary to build the system yourself to deploy or interact with Sleeper. In the future we may publish
+pre-built artefacts that will make this unnecessary.
 
-We'll use the Sleeper CLI to deploy an EC2 instance in an environment suitable for working with Sleeper.
+## Install Sleeper CLI
+
+The Sleeper CLI runs a Docker container that contains all the necessary tools to build and deploy Sleeper. This can give
+you a command line inside a container with these tools pre-installed, or run commands in such a container. This way you
+can avoid needing to install any dependencies other than Docker on your machine.
 
 ### Dependencies
 
-The Sleeper CLI has the following dependencies:
+The Sleeper CLI has the following dependencies, please install these first:
 
 * [Bash](https://www.gnu.org/software/bash/): Minimum v3.2. Use `bash --version`.
 * [Docker](https://docs.docker.com/get-docker/)
 
-### Install Sleeper CLI
-
-**Due to a bug in GitHub, only one version of the CLI is available.** See the following
-issue: https://github.com/gchq/sleeper/issues/2494
+### Install script
 
 You can run the following commands to install the latest version of the CLI from GitHub:
```

Comment:
After the bash box with the 3 commands in, can we add a step to check that the CLI has been installed correctly? I sometimes find that even after I've run those steps, and logged out of the box to get a new shell, typing sleeper results in "Command 'sleeper' not found"